src/components/Pagination/Pagination.tsx

```diff
@@ -1 +1,82 @@
-export const Pagination = () => {};
+import cn from 'classnames';
+
+type Props = {
+  total: number;
+  perPage: number;
+  currentPage: number;
+  onPageChange: (page: number) => void;
+  items: string[];
+};
+
+export const Pagination: React.FC<Props> = ({
+  total,
+  perPage,
+  currentPage,
+  onPageChange,
+  items,
+}) => {
+  const countTabs = Math.ceil(total / perPage);
+
+  const disablePrev = currentPage === 1;
+  const disableNext = currentPage === countTabs;
+
+  return (
+    <>
+      <ul className="pagination">
+        <li
+          className={cn('page-item', {
+            disabled: disablePrev,
+          })}
+        >
+          <a
+            data-cy="prevLink"
+            className={cn('page-link')}
+            href="#prev"
+            aria-disabled={disablePrev}
+            onClick={() => !disablePrev && onPageChange(currentPage - 1)}
+          >
+            «
+          </a>
+        </li>
+
+        {items.slice(0, countTabs).map((tab, index) => {
```

Comment:
```suggestion
        {getNumbers(1, countTabs).map(page => {
```
and remove the items prop at all